core/src/main/java/io/hyperfoil/core/impl/statistics/StatisticsStore.java

```diff
@@ -471,15 +474,15 @@ public void persist(Path dir) throws IOException {
       Data[] sorted = this.data.values().stream().flatMap(map -> map.values().stream()).toArray(Data[]::new);
       Arrays.sort(sorted, Comparator.comparing((Data d) -> d.phase).thenComparing(d -> d.metric).thenComparingInt(d -> d.stepId));
 
-      try (FileOutputStream stream = new FileOutputStream(dir + File.separator + "all.json")) {
-         JsonFactory jfactory = new JsonFactory();
-         jfactory.setCodec(new ObjectMapper());
-         JsonGenerator jGenerator = jfactory.createGenerator(stream, JsonEncoding.UTF8);
-         jGenerator.setCodec(new ObjectMapper());
+         try (FileOutputStream stream = new FileOutputStream(dir + File.separator + "all.json")) {
```

Comment:
This indent looks wrong

Reply:
fixed that too, odd that it passed checkstyles